src/PrestaShopBundle/Form/Admin/Catalog/Category/AbstractCategoryType.php

```diff
@@ -211,36 +278,65 @@ public function buildForm(FormBuilderInterface $builder, array $options)
+                'label' => $this->trans('Meta keywords', 'Admin.Global'),
+                'help' => $this->trans('To add tags, press the \'enter\' key. You can also use the \'comma\' key. Invalid characters: <>;=#{}', 'Admin.Shopparameters.Help')
+                    . '<br>' . $genericCharactersHint,
                 'required' => false,
                 'options' => [
                     'constraints' => [
-                        new Regex([
-                            'pattern' => '/^[^<>={}]*$/u',
-                            'message' => $this->trans('%s is invalid.', 'Admin.Notifications.Error'),
+                        new TypedRegex([
+                            'type' => TypedRegex::TYPE_GENERIC_NAME,
+                        ]),
+                        new Length([
+                            'max' => SeoSettings::MAX_KEYWORDS_LENGTH,
+                            'maxMessage' => $this->trans(
+                                'This field cannot be longer than %limit% characters.',
+                                'Admin.Notifications.Error',
+                                [
+                                    '%limit%' => SeoSettings::MAX_KEYWORDS_LENGTH,
+                                ]
+                            ),
                         ]),
                     ],
                     'attr' => [
+                        'maxlength' => SeoSettings::MAX_KEYWORDS_LENGTH,
                         'class' => 'js-taggable-field',
                         'placeholder' => $this->trans('Add tag', 'Admin.Actions'),
                     ],
                     'required' => false,
                 ],
             ])
             ->add('link_rewrite', TranslatableType::class, [
+                'label' => $this->trans('Friendly URL', 'Admin.Global'),
+                'help' => $this->trans('Allowed characters: letters, numbers, underscores (_) and hyphens (-). To allow more characters, enable the \'Accented URL\' feature in Shop Parameters > Traffic & SEO.', 'Admin.Catalog.Help'),
                 'type' => TextType::class,
                 'constraints' => [
                     new DefaultLanguage(),
                 ],
                 'options' => [
+                    'attr' => [
+                        'maxlength' => CategorySettings::MAX_TITLE_LENGTH,
+                    ],
                     'constraints' => [
-                        new Regex([
-                            'pattern' => (bool) $this->configuration->get('PS_ALLOW_ACCENTED_CHARS_URL') ? '/^[_a-zA-Z0-9\x{0600}-\x{06FF}\pL\pS-]+$/u' : '/^[^<>={}]*$/u',
-                            'message' => $this->trans('%s is invalid.', 'Admin.Notifications.Error'),
+                        new TypedRegex([
+                            'type' => TypedRegex::TYPE_LINK_REWRITE,
+                        ]),
+                        new Length([
+                            'max' => CategorySettings::MAX_TITLE_LENGTH,
```

Comment:
```suggestion
                            'max' => CategorySettings::MAX_LINK_REWRITE_LENGTH,
```

Reply:
I even think it should be in `SeoSettings` considering it seems to contain settings about Category SEO related fields already (the expected value is 128 by the way)